src/doc/reference.md

```diff
@@ -3064,6 +3064,17 @@ of a condition expression it expects a refutable let statement. If the value of
 expression on the right hand side of the let statement matches the pattern, the corresponding
 block will execute, otherwise flow proceeds to the first `else` block that follows.
 
+```
```

Comment:
Ah, I see. How about saying "This body will not execute because the pattern is refuted"